arrows/ocv/resection_camera.cxx

```diff
@@ -0,0 +1,218 @@
+// This file is part of KWIVER, and is distributed under the
+// OSI-approved BSD 3-Clause License. See top-level LICENSE file or
+// https://github.com/Kitware/kwiver/blob/master/LICENSE for details.
+
+/// \file
+/// \brief OCV resection_camera algorithm implementation
+
+#include "camera_intrinsics.h"
+#include "resection_camera.h"
+
+#include <opencv2/calib3d/calib3d.hpp>
+#include <opencv2/core/eigen.hpp>
+
+using namespace std;
+using namespace cv;
+
+namespace kwiver {
+
+namespace arrows {
+
+namespace ocv {
+
+// ----------------------------------------------------------------------------
+
+struct resection_camera::priv
+{
+  double reproj_accuracy = 4.;
+  int max_iterations = 32;
+  vital::logger_handle_t m_logger;
+
+  priv() : m_logger{ vital::get_logger( "arrows.ocv.resection_camera" ) }
+  {
+  }
+};
+
+// ----------------------------------------------------------------------------
+
+resection_camera
+::resection_camera()
+  : d_{ new priv }
+{
+}
+
+resection_camera::~resection_camera()
+{
+}
+
+// ----------------------------------------------------------------------------
+vital::config_block_sptr
+resection_camera
+::get_configuration() const
+{
+  // get base config from base class
+  vital::config_block_sptr config =
+    vital::algo::resection_camera::get_configuration();
+  config->set_value( "reproj_accuracy", d_->reproj_accuracy,
```

Comment:
BTW, do we have any conventions on config naming? Particularly, I'm wondering if this should be abbreviated...

Reply:
(Soliciting other thoughts from @mleotta in particular...)